examples/flutter_gallery/lib/demo/text_field_demo.dart

```diff
@@ -79,26 +112,19 @@ class TextFieldDemoState extends State<TextFieldDemo> {
             new Input(
               hintText: 'What do people call you?',
               labelText: 'Name',
-              formField: new FormField<String>(
-                // TODO(mpcomplete): replace with person#name=
-                setter:  (String val) { person.name = val; },
-                validator: _validateName
-              )
```

Comment:
> The selection is a property of the editor, not the data model.

This is an interesting observation. It's tricky because the selection is tied to the string in many ways. What would one expect to happen if we had two text fields backed by the same string, such that editing one changed the other — should the selection be mirrored in both? What happens if the selection in one becomes invalid because of edits in the other?